src/EA.Weee.RequestHandlers.Tests.Unit/Organisations/OrganisationByIdHandlerTests.cs

```diff
@@ -260,25 +265,33 @@ public async Task HandleAsync_GivenOrganisationHasNoDirectRegistrant_Organisatio
         [Fact]
         public async Task HandleAsync_GivenOrganisationHasDirectRegistrant_OrganisationShouldHaveDirectRegistrant()
         {
+            // Arrange
             var organisation = GetOrganisationWithId(organisationId);
-
             var expectedReturnValue = new OrganisationData();
             A.CallTo(() => map.Map(A<Organisation>._)).Returns(expectedReturnValue);
 
             var directRegistrant = A.Fake<DirectRegistrant>();
-            A.CallTo(() => directRegistrant.Id).Returns(Guid.NewGuid());
+            var directRegistrantId = Guid.NewGuid();
+
+            A.CallTo(() => directRegistrant.Id).Returns(directRegistrantId);
             A.CallTo(() => directRegistrant.OrganisationId).Returns(organisation.Id);
 
-            var directRegistrants = new List<DirectRegistrant> { directRegistrant };
+            A.CallTo(() => directRegistrant.DirectProducerSubmissions).Returns(new List<DirectProducerSubmission>
+            {
+                new DirectProducerSubmission { ComplianceYear = DateTime.UtcNow.Year }
```

Comment:
@will-hope98  We might get issues with the test running in new years. Think we need to setup the systemDataDataAccess to return a known date with compliance year  and use that as the compliance year.

Reply:
Done